packages/sophtron-adapter/scripts/rename-esm.sh

```diff
@@ -1,5 +1,7 @@
 #!/bin/bash
 
+sleep 5
```

Comment:
Why is this necessary?

Reply:
I found that sometimes this script errors when it runs, and the error is about files not existing. It seems the `tsc` build command that happens right before this script is run, needs a little more time for the files to show up in the file system before this can be run. I tried a couple other things, and this was the only one that was consistently successful.

Reply:
I've removed the sleep and I'll keep an eye on this to see if we continue to run into this issue.

Reply:
Still erroring. This is why I tried adding the sleep command: https://stackoverflow.com/a/78368373/97358